lib/sync/fetcher.go

```diff
@@ -174,15 +177,27 @@ func (f *BlockFetcher) fetch(ctx context.Context, si *SyncInfo) error {
 }
 
 // pickRandomNode choose one node by random. It is very protype for choosing fetching which node
-func (f *BlockFetcher) pickRandomNode() node.Node {
+func (f *BlockFetcher) pickRandomNode(nodeAddrs []string) node.Node {
```

Comment:
If https://github.com/bosnet/sebak/pull/495#issuecomment-428420886 is right, I wish `nodeAddrs` needs another understandable name, or you can explain what is `nodeAddrs` in comment.

Reply:
type SyncInfo struct {
	BlockHeight uint64
	Block       *block.Block
	Txs         []*transaction.Transaction

	NodeAddrs []string // Fetching target node addresses
}

It have already comments :-)

Reply:
@anarcher I wish this kind of things :)
```
NodeAddrs []string // Fetching target node addresses, `NodeAddrs` is  the validators which 
                   //participated and confirmed the consensus of latest ballot.
```

Reply:
@spikeekips Thanks a lot!! :->